web/filament-js/utilities.js

```diff
@@ -260,6 +260,14 @@ Filament._createTextureFromKtx = function(ktxdata, engine, options) {
 Filament._createIblFromKtx = function(ktxdata, engine, options) {
     options = options || {};
     const iblktx = options['ktx'] = new Filament.KtxBundle(ktxdata);
+
+    // We recommend using R11F_G11F_B10F for the IBL.
+    const internalFormat = iblktx.info().glInternalFormat;
+    if (internalFormat != 0x8c3a) {
```

Comment:
Constant instead of a magic number? :)

Reply:
RGB(A)16F is also a valid format btw.

Reply:
Yeah but cmgen uses 11_11_10 when outputting ktx, so it is very suspicious if a user isn't using that format.  I will fix the magic number and change the error to a warning.